evalml/data_checks/invalid_targets_data_check.py

```diff
@@ -59,25 +59,24 @@ def validate(self, X, y):
             raise ValueError("y cannot be None")
 
         y = _convert_to_woodwork_structure(y)
```

Comment:
Moved above because we can do logic with Woodwork types, before converting to pandas to determine # of null